app/cdap/components/ConnectorTypes/index.tsx

```diff
@@ -34,23 +34,28 @@ const ConnectorTypesComponent = () => {
 
   return (
     <>
-      <Paper variant="outlined" elevation={9} className={classes.dashBoard}>
-        <WelcomeCard />
-        <Typography className={classes.subTitle}>Start data exploration</Typography>
-        <Box className={classes.underLine}>{UnderLine}</Box>
-        <Paper elevation={0} className={classes.flexContainer}>
-          {connectorTypesList.map((eachConnectorType) => (
-            <Link
-              to={`/ns/${getCurrentNamespace()}/datasets-list/${eachConnectorType.name}`}
-              className={classes.linkLine}
-            >
-              <WranglerCard
-                key={eachConnectorType.name}
-                name={eachConnectorType.name}
-                image={eachConnectorType.image}
-              />
-            </Link>
-          ))}
+      <Paper className={classes.wrangleHomeContainer}>
+        <Paper variant="outlined" elevation={9} className={classes.dashBoard}>
+          <WelcomeCard />
+          <Typography className={classes.subTitle}>Start data exploration</Typography>
+          <Box className={classes.underLine}>{UnderLine}</Box>
+          <Paper elevation={0} className={classes.flexContainer}>
```

Comment:
Use MUI Grid Component for alignment of connector types component and on going exploarations component

Reply:
Added Grid to Connector Types Component.
Structure 
<Grid>
<Grid> //left side
</Grid>
<Grid>
</Grid> right side
</Grid>